alf/algorithms/mi_estimator.py

```diff
@@ -75,7 +75,7 @@ class MIEstimator(Algorithm):
        samples from the two buffers.
     * 'shuffle': randomly shuffle batch y
     * 'shift': shift batch y by one sample, i.e.
-      torch.cat([y[-1:, ...], y[0:-1, ...]], dim=0)
+      ``torch.cat([y[-1:, ...], y[0:-1, ...]], dim=0)``
```

Comment:
There are still lots of equations in this docstring that are not formatted in latex?